### PR TITLE
Toast

### PR DESCRIPTION
**Pull request by Gautam2305**
completed toast component with absolute positioning

Screenshot (94)

### CODE REVIEW

File: stylesheets/Components/toast.css [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+    position: relative;
+}
+.toast{
+    background-color: var(--st-darkest-grey);
+    color: var(--st-white);
+    max-width: 60%;
+    padding: 5px 20px 5px 20px;
+    border-radius: 4px;
+    position: absolute;
+    top: 30rem;
```
Comment (aishwarya-bm) [inline stylesheets/Components/toast.css:11]: Instead of giving top here you can give bottom

Comment (nishasen) [reply stylesheets/Components/toast.css:11]: Yes I also suggest this change

File: stylesheets/Components/toast.css [hunk 2]
```diff
@@ -0,0 +1,13 @@
+.toast-container{
+    position: relative;
+}
+.toast{
+    background-color: var(--st-darkest-grey);
+    color: var(--st-white);
+    max-width: 60%;
+    padding: 5px 20px 5px 20px;
```
Comment (nishasen) [inline stylesheets/Components/toast.css]: Use shorthand for this
padding : 5px 20px;

File: stylesheets/Components/toast.css (hunk 1 above)
Comment (Gautam2305) [reply stylesheets/Components/toast.css:11]: okay

File: stylesheets/Components/toast.css (hunk 2 above)
Comment (Gautam2305) [reply stylesheets/Components/toast.css]: yeah..okay ...will do that

File: stylesheets/Components/toast.css (hunk 1 above)
Comment (Gautam2305) [reply stylesheets/Components/toast.css:11]: but it will go further up in the screen right.. because initially its at the top of the screen

**Pull request merged**